### ISSUE TITLE
The minecraft:hoes tag fails to load

### ISSUE THREAD

**Issue by ChiefArug** [MrManiacc/Synthesize#5]
### Describe the bug?

This snippet from the log explains it pretty well: 
```
[Worker-Main-47/ERROR] [net.minecraft.tags.TagLoader/]: Couldn't load tag minecraft:hoes as it is missing following references: mekanismtools:bronze_hoe (from synth-1.18.2-04-28.jar), 
	mekanismtools:lapis_lazuli_hoe (from synth-1.18.2-04-28.jar), 
	mekanismtools:osmium_hoe (from synth-1.18.2-04-28.jar), 
	mekanismtools:refined_glowstone_hoe (from synth-1.18.2-04-28.jar), 
	mekanismtools:refined_obsidian_hoe (from synth-1.18.2-04-28.jar), 
	mekanismtools:steel_hoe (from synth-1.18.2-04-28.jar), 
	mysticalagriculture:inferium_hoe (from synth-1.18.2-04-28.jar), 
	mysticalagriculture:prudentium_hoe (from synth-1.18.2-04-28.jar), 
	mysticalagriculture:tertium_hoe (from synth-1.18.2-04-28.jar), 
	mysticalagriculture:imperium_hoe (from synth-1.18.2-04-28.jar), 
	mysticalagriculture:supremium_hoe (from synth-1.18.2-04-28.jar)
```
tldr: If you do not have all of the above mods installed the entire tag `minecraft:hoes` will fail to load, causing any recipes or mods that depend on it to also fail.

### Relevant issues

No, and why is this a required field?

### To Reproduce

Load the mod on its own.
Look in the log, and check in game. 
The error will show up in the log and the tag is not applied in game.

### Expected behavior

Not to add hoes from other random mods, causing the entire tag to fail to load. 

### Screenshots

Using KubeJS to check the items in the minecraft hoes tag: 
![image](https://user-images.githubusercontent.com/73862885/167085914-501cd51f-9292-46c8-9e10-dd2897d00d4e.png)

### How often does it happen?

Always

### Does the bug persist after restarting?

Yes

### Mod version number

synth-1.18.2-04-28

### Did this work before?

No

### Additional context

The file that is causing it: https://github.com/synth-dev/synth/blob/main/src/main/resources/data/minecraft/tags/items/hoes.json

**Comment by MrManiacc** [comment]
Thank you for reporting this. I'll make sure to check to see if mekanism is loaded before I push the 1.19 update.
